meterfs/meterfs.c

```diff
@@ -1311,3 +1312,146 @@ int meterfs_init(meterfs_ctx_t *ctx)
 
 	return 0;
 }
+
+
+/* TODO Change meterfs interfaces to natively support libstorage. */
+/* Temporary adapters to implement storage fsops interfaces */
```

Comment:
Move these wrappers to a separate file (along with mount/unmount), create new header for this API (mount and unmount only should be exported)